pkg/cli/mirror/catalog_images.go

```diff
@@ -558,3 +480,78 @@ func deleteLayer(old string) (v1.Layer, error) {
 	deleteMap[old] = []byte{}
 	return crane.Layer(deleteMap)
 }
+
+func copyOPMBinary(img v1.Image, target string) error {
+	cfgf, err := img.ConfigFile()
+	if err != nil {
+		return err
+	}
+	entrypoint := cfgf.Config.Entrypoint
+	opmBin := strings.TrimPrefix(entrypoint[0], (string)(os.PathSeparator))
```

Comment:
are we confident of the fact that there will always be an entrypoint[0] - should we check first and error if we don't have the correct length ?